Storage/StorageTests/CoreData/MigrationTests.swift

```diff
@@ -3017,6 +3017,34 @@ final class MigrationTests: XCTestCase {
         XCTAssertEqual(try targetContext.count(entityName: "MetaData"), 2)
         XCTAssertEqual((migratedOrder.value(forKey: "customFields") as? NSSet)?.count, 2)
     }
+
+    func test_migrating_from_115_to_116_adds_new_startTime_attribute_to_BlazeCampaignListItem() throws {
+        // Given
+        let sourceContainer = try startPersistentContainer("Model 115")
+        let sourceContext = sourceContainer.viewContext
+
+        let campaign = insertBlazeCampaignListItem(to: sourceContext)
+        try sourceContext.save()
+
+        // Confidence check: new startTime attribute is not present
+        XCTAssertNil(campaign.entity.attributesByName["startTime"])
```

Comment:
Just as a double check (because otherwise this can fake the test), it would be great to make sure that the inserted `BlazeCampaignListItem` has been saved. What do you think?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/13807/commits/5d8263ddcee4ee8a58c7443f54b47e1b8c308989